src/index.html

```diff
@@ -5,8 +5,336 @@
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link header__nav_link--is-active is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a
+              href="#"
+              class="header__nav_link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Photo
+            </a>
+          </li>
+          <li class="header__nav_item">
+            <a href="#" class="header__nav_link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__image"
+          >
+          <div class="card__text">
+            <p class="card__descriprion">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__first-row">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </div>
+            <div class="card__second-row">
+              <p class="card__price">Price:</p>
+              <p class="card__amount">$2,199</p>
+            </div>
+            <div class="card__button">
+              <a
+                href="#"
+                class="card__link card__link--active"
+                data-qa="card-hover"
+              >
+                BUY
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="card catalog__card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__image"
+          >
+          <div class="card__text">
```

Comment:
if you want use container for some product informtion, please provide more descriptive class name, for example card__product-info